test/integration/executing_test.exs

```diff
@@ -71,7 +71,8 @@ defmodule Oban.Integration.ExecutingTest do
 
   defp job do
     gen all queue <- member_of(~w(alpha beta gamma delta)),
-            action <- member_of(~w(OK DISCARD ERROR EXIT FAIL KILL SNOOZE TASK_ERROR)),
+            action <-
+              member_of(~w(OK DISCARD ERROR EXIT FAIL KILL SNOOZE TASK_ERROR TASK_EXIT_ERROR)),
```

Comment:
The other action is `EXIT`, not `EXIT_ERROR` (and this keeps it on a single line still).

```suggestion
            action <- member_of(~w(OK DISCARD ERROR EXIT FAIL KILL SNOOZE TASK_ERROR TASK_EXIT),
```